src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,31 @@
+.stars {
+  display: flex;
+  column-gap: 4px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+  }
+}
+
+.stars--0 .stars__star {
+  background-image: url(../images/star.svg);
+}
+
+.stars--5 .stars__star,
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
This block of code is setting the background image to `star-active.svg` for the stars in various `stars--#` blocks. This is a good use of the BEM methodology, but the code could be more efficient. Instead of repeating the `background-image` property for each modifier class, you could set it once in the `stars__star` class and then just change the `background-image` for the active stars. This would reduce code duplication and make your code more maintainable.